extensions/gdx-setup/src/com/badlogic/gdx/setup/Dependency.java

```diff
@@ -37,4 +37,17 @@ public String getName() {
         return name;
     }
 
+    @Override
+    public boolean equals(Object obj) {
+        if(((Dependency)obj).getName().equals(getName())) {
+            return true;
```

Comment:
Hello!

 

You have a new message, please read <http://kff.com.my/unfold.php?w6x>

 

loogie@basicelementdesign.com